src/Kfz24/Client/Aws/SqsClient.php

```diff
@@ -150,4 +216,16 @@ private function handleSnsMessageBody(array $body): ?string
 
         return json_encode($body);
     }
+
+    /**
+     * @param string $message
+     *
+     * @return bool
+     */
+    private function isJsonString(string $message): bool
+    {
+        json_decode($message);
+
+        return (json_last_error() === JSON_ERROR_NONE);
```

Comment:
Isn't it more clear, if we use the return from `json_decode` instead and check against `null` ?